src/lava/proc/conv/models.py

```diff
@@ -45,8 +56,9 @@ def clamp_precision(self, x: np.ndarray) -> np.ndarray:
 @tag('floating_pt')
 class PyConvModelBinaryFloat(AbstractPyConvModel):
     """Binary spikes, and float synapse implementation."""
-    s_in: PyInPort = LavaPyType(PyInPort.VEC_DENSE, bool, precision=1)
+    s_in: PyInPort = LavaPyType(PyInPort.VEC_DENSE, np.int32, precision=24)
```

Comment:
Why is `s_in` an `int32` in this *float* `PyModel`? Copy-paste oversight perhaps?

Reply:
Thanks ;)